packages/codemods/README.md

```diff
@@ -0,0 +1,83 @@
+# ts-morphin-migration
+
+ts-morphin-migration is a package created and managed with the `create-just` utility.
+
+## Next Steps
+
+Run
+
+```sh
+yarn
+yarn start-test
+```
+
+To start testing the codemods
```

Comment:
In general I think the description of what the commands do should go above the commands not below